modules/javafx.fxml/src/main/java/javafx/fxml/FXMLLoader.java

```diff
@@ -1098,9 +1098,15 @@ public void processAttribute(String prefix, String localName, String value)
                     if (loadListener != null) {
                         loadListener.readInternalAttribute(localName, value);
                     }
-
-                    resources = ResourceBundle.getBundle(value, Locale.getDefault(),
-                            FXMLLoader.this.resources.getClass().getClassLoader());
+                    final ResourceBundle loaderResources = FXMLLoader.this.resources;
```

Comment:
Why to have this local temporary `loaderResources`?
We can simply use `this.resources` for readability.

Reply:
Done.